src/fish/point.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable camelcase */
+/* eslint-disable no-undef */
+import {getRandomInteger} from '../util.js';
+
+const generateTypePoint = () => {
+  typesPoint = [
```

Comment:
Перед использованием нужно инициализировать переменные, например const typesPoint или let typesPoint
```suggestion
  const typesPoint = [
```